ais/api/views.py

```diff
@@ -395,18 +477,14 @@ def search_view(query):
                 'include_units' in request.args,
                 is_range=high_num is not None,
                 is_unit=unit_type is not None) \
-                .exclude_non_opa('opa_only' in request.args)
+                .exclude_non_opa('opa_only' in request.args) \
```

Comment:
this is the general search endpoint - addresses are being queried under the elif search_type = "address" section